source/simulator/solver_schemes.cc

```diff
@@ -291,8 +291,9 @@ namespace aspect
         const double relative_stokes_residual =
           assemble_and_solve_stokes(nonlinear_iteration == 0, &initial_stokes_residual);
 
-        pcout << "      Relative Stokes residual after nonlinear iteration " << nonlinear_iteration+1
+        pcout << "      Relative  nonlinear residual (Stokes system only) after nonlinear iteration " << nonlinear_iteration+1
```

Comment:
Is it important to differentiate between `Stokes system` and `Stokes system only`? The residual is calculated the same way after all.

Reply:
No, I guess it's not. I thought it would be a nice way to remind the user of the solver scheme used, but I guess that's easy to see anyway due to the equations that are solved. So should I just call it Stokes system in both cases?

Reply:
Yes that seems more consistent.